packages/frameworks/solid-vite/package.json

```diff
@@ -45,33 +45,23 @@
     "*.d.ts"
   ],
   "scripts": {
-    "build": "node --loader ts-node/esm bundle.ts",
-    "install-local": "yarn install --registry http://localhost:6001 --no-lockfile",
-    "publish-local": "npm publish --registry http://localhost:6001",
-    "unpublish-local": "npm unpublish storybook-solidjs-vite@1.0.0-beta.2 --registry http://localhost:6001"
+    "prepack": "yarn prep",
+    "prep": "npx jiti ../../../scripts/prepare/bundle.ts",
+    "check": "npx jiti ../../../scripts/prepare/check.ts"
   },
   "dependencies": {
-    "@storybook/builder-vite": "next"
+    "@storybook/builder-vite": "^8.3.5",
```

Comment:
We should probably be working against `next` for now. I n order to get this all merged & released we will need to jump through a few hoops because we test the `solid-vite` framework in the storybook monorepo and i don't want this to block mainline development. I would probably keep it on `next`, but it if it's important for you to get this onto `latest`, we can do that in the separate PR after we've gotten this merged.

Reply:
Hmm, is this for backwards compatibility? From what it sounds like, the release testing is something along the lines of `pull latest storybook-solid -> run CLI -> check for errors`, and you need it to use `next` so it pulls the latest release candidate, right?

My concern is that by using `next`, using an old version of `storybook-solid` would break if you try to update dependencies to latest minor/patch. It's not a big deal to me right now, so I'll move it to `next`, but in the future, I would like to refactor it so it can use `next` for testing and a concrete semantic version for release.